codes/c/include/array_hash_map.h

```diff
@@ -0,0 +1,74 @@
+/**
+ * File: hash_map.h
+ * Created Time: 2023-03-18
+ * Author: Guanngxu (446678850@qq.com)
```

Comment:
Placing this file in the same folder with `array_hash_map.c` would be a better choice because it is not the shared library.